locale/en/site.json

```diff
@@ -163,15 +163,10 @@
   },
   "banners": {
     "index": {
-      "startDate": "2022-01-04T16:00:00.000Z",
-      "endDate": "2022-01-25T16:00:00.000Z",
-      "text": "New security releases now available for Node.js 12, 14, 16, and 17 release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/jan-2022-security-releases/"
-    },
-    "blacklivesmatter": {
```

Comment:
Just update the index banner, not this one

Reply:
This removes that banner info from site.json, which I think is OK, but if you want/need it in a separate PR, I guess we could do that too.

Reply:
@nschonni just pushed a change to limit to the right part, look ok?